### PR TITLE
[Android] Update android dependencies

### PR DESCRIPTION
**Pull request by talregev**
Update android dependencies.
Compile, and run and crunch on One plus One (android 9) and emulator Arm7 (android 4.1)

### CODE REVIEW

File: android/BOINC/.idea/misc.xml [hunk 1]
```diff
@@ -45,5 +45,5 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_X" project-jdk-name="11" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_16" project-jdk-name="11" project-jdk-type="JavaSDK" />
```
Comment (AenBleidd) [inline android/BOINC/.idea/misc.xml:48]: This file is autogenerated and should be removed from the repo at all. Why did you make this change here?

File: android/BOINC/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -17,7 +17,6 @@
   along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-        package="edu.berkeley.boinc"
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/AndroidManifest.xml:20]: Why did you remove this?

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: android/BOINC/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (talregev) [reply android/BOINC/app/src/main/AndroidManifest.xml:20]: Android studio suggest to remove it from here and add it on build.gradle. I didn't object to that suggestion.

File: android/BOINC/.idea/misc.xml (hunk 1 above)
Comment (talregev) [reply android/BOINC/.idea/misc.xml:48]: We have partial android studio project in our repo. In first I was object but now I understand the advantage of that. That it faster to load your android studio and it similar dev for all dev.

File: android/BOINC/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (talregev) [reply android/BOINC/app/src/main/AndroidManifest.xml:20]: It did automated autogenerated suggestion

**Comment by talregev** [comment]
I am putting this PR on hold. I want to test more.

**Comment by talregev** [comment]
I tested it again couple time and check against the master, and I see that we good to go.
@AenBleidd Do you want to test it local also from your side?

**Comment by talregev** [comment]
Thank you!
